trunk/source/WebsiteStatic/build_all_settings.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+
+import os
+from build_settings import compute_settings
+
+SCRIPT_DIR  = os.path.dirname(__file__)
+EPF_ROOT    = os.path.join(SCRIPT_DIR, '../../examples/settings/')
+WORKER_ROOT = os.path.join(SCRIPT_DIR, 'webinterface/workers/')
+
+
+WORKER_SETTINGS = {
```

Comment:
They should match; changes since the PR opened would have lead to conflicts, in which case I did change the epf file (e.g. disabling the live-cfg for Referee).